deployments/helm-chart/README.md

```diff
@@ -194,7 +195,7 @@ Parameter | Description | Default
 `controller.nginxDebug` | Enables debugging for NGINX. Uses the `nginx-debug` binary. Requires `error-log-level: debug` in the ConfigMap via `controller.config.entries`. | false
 `controller.logLevel` | The log level of the Ingress Controller. | 1
 `controller.image.repository` | The image repository of the Ingress controller. | nginx/nginx-ingress
-`controller.image.tag` | The tag of the Ingress controller image. | edge
+`controller.image.tag` | The tag of the Ingress controller image. | 1.9.0
```

Comment:
deployments/helm-chart/values-icp.yaml , deployments/helm-chart/values.yaml and deployments/helm-chart/values-plus.yaml need to be updated as well